MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,93 @@
+   {% endcolumn}
+   {% column %}
+   ![yearview-in-maui-calendar](images/overview/MultipleViews/yearview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![decadeview-in-maui-calendar](images/overview/MultipleViews/decadeview-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-in-maui-calendar](images/overview/MultipleViews/centuryview-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the different view modes in the SfCalendar.
+
+* **Allow view Navigation**: Allows you to navigate to the different view modes through tap interaction on the cell or header when it is enabled. Restrict users from navigating to differnt calendar views and allows you to select the dates in the year, decade and century view cells when this property is not enabled.
+
+   {% multiple_column %}
+   {% column %}
+   ![monthview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/monthview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![yearview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/yearview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![decadeview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/decadeview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![centuryview-range-selection-in-maui-calendar](images/overview/SelectionWhileAllowViewNavigation/centuryview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Enable Past Dates**: Indicates whether the dates are enabled or disabled before the today date. If this property is disabled, you cannot able to select the dates before the today date and the dates before the today date are considered as disabled dates.
+
+   ![enable-past-date-in-maui-calendar](images/overview/enable-past-date-in-maui-calendar.png)
+
+* **Date selection**: The Selection mode is specified in the calendar property enumeration, allows you to select Single, Multiple, and a Range of dates through interaction. It also supports programmatic selection.
+
+   {% multiple_column %}
+   {% column %}
+   ![monthview-single-selection-in-maui-calendar](images/overview/DateSelection/monthview-single-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   ![monthview-multiple-selection-in-maui-calendar](images/overview/DateSelection/monthview-multiple-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% column %}
+   !![monthview-range-selection-in-maui-calendar](images/overview/DateSelection/monthview-range-selection-in-maui-calendar.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Limit the date selection range**: You can select a date from a range of dates between a minimum and maximum number of days by setting the minimum and maximum dates (span of days).
```

Comment:
Improve content 

"maximum number of days"

Reply:
Improved the Limit the date selection range content and explained about the minimum and maximum number of days